src/cls/queue/cls_queue_src.cc

```diff
@@ -27,6 +27,11 @@ int queue_write_head(cls_method_context_t hctx, cls_queue_head& head)
 
   bl.claim_append(bl_head);
 
+  if (bl.length() > head.max_head_size) {
+    CLS_LOG(0, "ERROR: queue_write_head: invalid head size = %u and urgent data size = %u \n", bl.length(), head.bl_urgent_data.length());
```

Comment:
Matt found that this log message has carriage returns in place of spaces between each word, so I need to clean this up. Hence I have added the DNM label